src/nplinker/config.py

```diff
@@ -10,6 +11,7 @@
 # directory.
 # The loaded config data is available by importing this module and accessing the 'config' variable.
 
+__all__ = ["config"]
```

Comment:
Why is this needed?

Reply:
To make sure only the variable `config` is imported when using `from nplinker.config import *`. We don't want the other variables like `validators` imported.